lru_cache.py

```
@@ -1,58 +1,28 @@
-    使用 OrderedDict 实现，保持 O(1) 的访问和插入时间复杂度
+    使用 OrderedDict 记录键的访问顺序, 新访问的放到末尾, 淘汰最早（最久未访问）的。
     """
-
-    def __init__(self, capacity: int):
-        """
-        初始化 LRU 缓存
-
-        :param capacity: 缓存容量
-        """
-        self.capacity = capacity
+    def __init__(self, capacitty: int):  # 问题1：参数名拼写错误 (多了t)
```

Ruff (0.14.6)

Unused method argument: `capacitty`

(ARG002)